llumnix/queue/zmq_utils.py

```diff
@@ -44,3 +44,7 @@ class RPCClientClosedError(Exception):
 
 def get_open_zmq_ipc_path(ip, port) -> str:
     return "tcp://{}:{}".format(ip, port)
+
+# generate zmq pool name with ip and port
```

Comment:
it's just a string defined by you own, maybe there is no need to add a seperate function?